numpy/lib/function_base.py

```diff
@@ -2325,6 +2325,11 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
         weights can be used to assign probabilities to observation vectors.
 
         .. versionadded:: 1.10
+    dtype : data-type, optional
+        Data-type of the result. By default, the return data-type is
+        `numpy.float64`.
```

Comment:
I am happy if we put this in. Technically, the default has at least `numpy.float64` precision (mostly, it could be `object` or `longdouble`).  I am a bit unsure if explaining that makes it just harder to understand though.